### PR TITLE
fixed unwanted dataset download

### PR DESCRIPTION
**Pull request by anandhu-eng**
In `get-dataset-imagenet-val` the path was initially compared with `CM_DATASET_PATH` which was only assigned  in the `postprocess` of `customize.py`. 
SInce the higher order of execution of cm script follows preprocess->prehook dep->run.sh/run.bat->postprocess, the value of `CM_DATASET_PATH` was always `NIL` which made the script to download the `ILSVRC2012_img_val_500.tar` from dropbox even if user had provided the path to dataset.

### CODE REVIEW

**Review by arjunsuresh — APPROVED** [review]
LGTM

**Comment by anandhu-eng** [comment]
The latest commit fixed the following issue:

image

File: cm-mlops/script/get-mlperf-inference-src/run.bat [hunk 1]
```diff
@@ -14,13 +14,11 @@ if not exist %folder% (
        git clone %CM_GIT_RECURSE_SUBMODULES% -b "%CM_GIT_CHECKOUT%" %CM_GIT_URL% %CM_GIT_DEPTH% %folder%
        IF %ERRORLEVEL% NEQ 0 EXIT %ERRORLEVEL%
        cd %folder%
+       git checkout "%CM_GIT_SHA%"
   ) else (
-       git clone %CM_GIT_RECURSE_SUBMODULES% %CM_GIT_URL% %CM_GIT_DEPTH% %folder%
+       git clone %CM_GIT_RECURSE_SUBMODULES% -b "%CM_GIT_CHECKOUT%" %CM_GIT_URL% %CM_GIT_DEPTH% %folder%
```
Comment (arjunsuresh) [inline cm-mlops/script/get-mlperf-inference-src/run.bat:19]: Actually we should use run.bat inside the get-git-repo script and delete this file. There, branch part is processed inside the customize.py and run.bat just needs to be made in sync with run.sh

https://github.com/mlcommons/ck/blob/master/cm-mlops/script/get-git-repo/customize.py

Comment (anandhu-eng) [reply cm-mlops/script/get-mlperf-inference-src/run.bat:19]: Thanks @arjunsuresh, I think apart from submodule initialization and patching, basic git commands are implemented in run.bat of `get-git-repo`. Do we have need the patching part  for running the inference models?

Comment (arjunsuresh) [reply cm-mlops/script/get-mlperf-inference-src/run.bat:19]: No. We don't need it.

**Review by arjunsuresh — APPROVED** [review]
LGTM. Thank you for the changes

**Pull request merged**